tracing-subscriber/src/registry/mod.rs

```diff
@@ -241,17 +316,32 @@ where
         })
     }
 
+    /// Returns an iterator over all parents of this span, starting with this span,
+    /// ordered from leaf to root.
+    ///
+    /// The iterator will first return the span, then the span's immediate parent,
+    /// followed by that span's parent,  and so on, until it reaches a root span.
+    pub fn scope(&self) -> Scope<'a, R> {
```

Comment:
it might be worth having the docs for this method also mention the existence of the `from_root` method --- ideally with an example?

Reply:
Done

Reply:
Ahh, sorry, misunderstood. I've made an attempt now, but I don't think it's very good (downcasting dispatchers, and a fairly unrealistic context).

Reply:
235ab8980279038b5dce613c1ba02e254929b2bc rewrites the examples to be in a layer context. The assertions are still kept around, but unrendered.